textBasedAdventure.js

```diff
@@ -1,5 +1,88 @@
-const readline = require('readline-sync')
 
+const readline = require('readline-sync')
 let nameInput = readline.question("Enter your name: ")
+console.log("Hello " + nameInput + " !  Welcome to my game, Survive.")
+const play = () => {
+    let userInput;
+    while (userInput !== "yes") {
+        userInput = readline.question("Would you like to play? yes or no?")
+        if (userInput === "no") {
+            console.log("Come on, it's just one game...")
+            userInput = readline.question("Are you going to play? okay or no?")
+            break;
+        } else {
+            console.log("Glad I could convince you.")
```

Comment:
You would want to `break` when the user picks yes. As it is now this would just end the game if they pick no.